README.md

```diff
@@ -1,37 +1,44 @@
 #  CASL Ansible
 
-Automation of OpenShift 3 using [Ansible](http://www.ansible.com/)
+Automation of OpenShift 3 using [Ansible](http://www.ansible.com/).
+(This includes automation of OpenShift Cluster provisioning as well as other automation tasks post-provisioning.)
 
-## Provisioning Quickstart
+## Provisioning (Quickstart)
 
 ### Local Setup (one time, only)
 
+> **NOTE:** These steps are a canned set of steps serving as an example, and may be different in your environment.
+
 In addition to _cloning this repo_, you'll need the following:
 
-* Access to an OpenStack environment using an [OpenStack RC File](http://docs.openstack.org/user-guide/common/cli-set-environment-variables-using-openstack-rc.html)
-  * File should be placed at `~/.config/openstack/openrc.sh`
->**NOTE**: OpenStack environment currently requires Heat to be enabled, and the user must have the `heat_stack_owner` role assigned.
-* A [Key-pair created in OpenStack](https://github.com/naturalis/openstack-docs/wiki/Howto:-Creating-and-using-OpenStack-SSH-keypairs-on-Linux-and-OSX)
-* Docker installed (`yum install -y docker` on RHEL/Centos, `dnf install docker -y` on Fedora)
-  * If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
-* Clone this repository (make sure to adjust paths as fit for your local environment - in this case `~src` is used):
+* Access to a Cloud Provider (OpenStack supported for the time being)
+* Docker installed
+  * RHEL/CentOS: `yum install -y docker`
+  * Fedora: `dnf install -y docker`
+  * **NOTE:** If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+* Specific Cloud Provider requirements (see below).
+* Clone this repository (make sure to adjust paths as fit for your local environment - in this case `~/src` is used):
+
 ```
 cd ~/src/
-git clone git@github.com:redhat-cop/casl-ansible.git
+git clone https://github.com/redhat-cop/casl-ansible.git
 ```
-* Copy `casl-ansible/inventory/sample.casl.example.com.d/inventory/clouds.yaml` to `~/.config/openstack/clouds.yaml`
 
-* Download/untar `openshift-ansible` (or use git clone) for use as part of the install. Make sure to do so within the same directory as above (i.e.: `~/src`), and either rename the directory to `openshift-ansible` or create symlink to it (see example below). See table below for versions / urls to be used for the download. Note that other versions may work as well, but these are the ones we have tested and found to be stable. Feel free to submit PRs for this file with updated versions as they are found to be functional.
+* Download/untar `openshift-ansible` for use as part of the install. Make sure to do so within the same directory as above (i.e.: `~/src`), and either rename the directory to `openshift-ansible` (or create symlink to it - see example below). See table below for versions / urls to be used for the download.
```

Comment:
Why did we remove the git clone portion? In my mind we should be pushing people more towards `git clone/git checkout release-3.x` than downloading tars.

Reply:
Maybe more of a preference;  I believe going the route of downloading a tar-file is "simpler" + allows for an easier way of using a particular release rather than always the "latest" 3.x.y.